src/microbithal_js.c

```diff
@@ -454,13 +454,7 @@ void microbit_hal_audio_select_speaker(bool enable) {
 
 // Input value has range 0-255 inclusive.
 void microbit_hal_audio_set_volume(int value) {
-    /*
-    if (value >= 255) {
-        uBit.audio.setVolume(128);
-    } else {
-        uBit.audio.setVolume(value / 2);
-    }
-    */
```

Comment:
Copying this flow doesn't seem to work well for the sim?

Reply:
We've put this on the agenda to discuss with Damien.
MakeCode appears to use the full volume range here.